src/styles/items.styl

```diff
@@ -1,81 +1,82 @@
-.item
```

Comment:
Sorry for the ugly diff, but this just adds an `#items` parent selector to everything in here so that it applies only to the view all items page.